graphql/resolvers/competition/queries.js

```diff
@@ -0,0 +1,11 @@
+import { Competition } from "$/db/index.js";
+
+export default {
+    competition: async (_, args) => {
+        return Competition.find(args);
+    },
+
+    competitions: async (_, args) => {
+        return Competition.find(args);
+    },
```

Comment:
These functions have exactly the same body and parameters. Why?

Reply:
Because they resolve different queries in the same way. ``args`` is actually different structure in both, and they also have different returns: single value and array. This should be more clear now.